lpr.Data/PackageData.cs

```diff
@@ -19,12 +19,9 @@ public class PackageData : IPackageData {
       return await _ctx.Package.Skip(page * amount).Take(amount).ToListAsync();
     }
 
-    public async Task<List<Package>> GetTopPackagesAsync(int amount) {
+    public Task<List<Package>> GetTopPackagesAsync(int amount)
+    {
       throw new NotImplementedException();
-      // TODO: Because Downloads are based on the collective downloads of the
```

Comment:
Maybe not removing comments from others? TODO comments can be helpful.

Reply:
Yea true, You are right. 
But to be honest, I don't like it that we are making methods for things that can't be implemented for while. 
We haven't even made a download option. or a create version endpoint. 

It's good thinking. But in this way, we will have an application with a lot of code that doesn't do anything. 
I am just afraid that it will be a mess code-wise.  
We will discuss this tomorrow 💯

Reply:
Yea true, You are right.
But to be honest, I don't like it that we are making methods for things that can't be implemented for while.
We haven't even made a download option. or a create version endpoint.

It's good thinking. But in this way, we will have an application with a lot of code that doesn't do anything.
I am just afraid that it will be a mess code-wise.
We will discuss this tomorrow 💯